pages/index.tsx

```diff
@@ -35,13 +36,15 @@ const FinanceOverviewPage: NextPage<FinanceOverviewPageProps> = ({
   }
 
   return (
-    <FinancesOverviewContainer
-      quarterExpenses={quarterExpenses}
-      monthlyExpenses={monthlyExpenses || []}
-      byBudgetBreakdownExpenses={byBudgetBreakdownExpenses}
-      byCategoryBreakdownExpenses={byCategoryBreakdownExpenses}
-      expenseCategories={expenseCategories}
-    />
+    <ModalCategoriesProvider expenseCategories={expenseCategories}>
+      <FinancesOverviewContainer
+        quarterExpenses={quarterExpenses}
+        monthlyExpenses={monthlyExpenses || []}
+        byBudgetBreakdownExpenses={byBudgetBreakdownExpenses}
+        byCategoryBreakdownExpenses={byCategoryBreakdownExpenses}
+        expenseCategories={expenseCategories}
```

Comment:
if you passed the `expenseCategories` in the provider, do you need to pass it as props to this component?

Reply:
You are right, It's not need it. Done